tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/LangLibFunctionTest.java

```diff
@@ -288,11 +289,11 @@ public void testRecordLangLib() {
         TypeSymbol type = ((TypeDefinitionSymbol) symbol).typeDescriptor();
         assertEquals(type.typeKind(), RECORD);
 
-        List<String> expFunctions = List.of("length", "iterator", "get", "entries", "'map", "forEach", "filter",
-                                            "reduce", "removeIfHasKey", "remove", "removeAll", "hasKey", "keys",
-                                            "toArray", "clone", "cloneReadOnly", "cloneWithType", "isReadOnly",
-                                            "toString", "toBalString", "toJson", "toJsonString", "fromJsonWithType",
-                                            "mergeJson", "ensureType");
+        List<String> expFunctions = List.of("length", "iterator", "get", "entries", "'map", "forEach",
+                "filter", "reduce", "removeIfHasKey", "remove", "removeAll", "hasKey", "keys",
+                "toArray", "clone", "cloneReadOnly", "cloneWithType", "isReadOnly",
+                "toString", "toBalString", "toJson", "toJsonString", "fromJsonWithType",
+                "mergeJson", "ensureType");
```

Comment:
Unnecessary formatting change is included

Reply:
> Unnecessary formatting change is included

Had got opinions from a few developers. It is okay to stick with the default formatting the code-editor does

Reply:
Got it.